src/libprojectM/ProjectM.hpp

```diff
@@ -91,8 +91,15 @@ typedef enum
 class ProjectM
 {
 public:
-    static const int FLAG_NONE = 0;
-    static const int FLAG_DISABLE_PLAYLIST_LOAD = 1 << 0;
+    /*
+     * Behaviour flags for the projectM instance. Currently, it's only used to prevent automatically filling
+     * the preset playlist by traversing the preset path for files.
+     */
+    enum Flags
+    {
+        None = 0, //!< No special flags.
+        DisablePlaylistLoad = 1 << 0 //!< Prevent automatic playlist loading on startup.
```

Comment:
Only one flag? Perhaps we should just use a boolean instead.

Reply:
Totally agree. Probably was meant to get more behavior flags at some point, but since it didn't, it's now definitely overengineered.